__tests__/app.test.js

```diff
@@ -133,102 +131,92 @@ describe("PATCH /api/articles/:article_id", () => {
-});
-
-test("400: Returns error message when no inc amount entered", () => {
-  const article_id = 2;
-  return request(app)
-    .patch(`/api/articles/${article_id}`)
-    .send({ inc_votes: '' })
-    .expect(400)
-    .then(({ body }) => {
-      expect(body.message).toBe('Could not update, please ensure you have entered a vote amount in number format');
-    })
-});
-
-});
-
+  test("404: Returns error message when invalid inc entered", () => {
+    const article_id = 2;
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send({ inc_votes: "ten" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          "Could not update, please ensure you have entered a vote amount in number format"
+        );
+      });
+  });
 
-describe('GET /api/articles', () => {
-  test('responds with an array of articles in descending date order', () => { 
+  test("400: Returns error message when no inc amount entered", () => {
+    const article_id = 2;
     return request(app)
-    .get("/api/articles")
-    .expect(200)
-    .then(({ body }) => {
-      // console.log(body)
-      let articles = body;
-      articles.forEach((article) => {
-        expect(article).toEqual(
-          expect.objectContaining({
-            article_id: expect.any(Number),
-            author: expect.any(String),
-            body: expect.any(String),
-            created_at: expect.any(String),
-            title: expect.any(String),
-            topic: expect.any(String),
-            votes: expect.any(Number),
-            comment_count: expect.any(String),
-          })
+      .patch(`/api/articles/${article_id}`)
+      .send({ inc_votes: "" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          "Could not update, please ensure you have entered a vote amount in number format"
         );
       });
-      expect(articles.length).toEqual(12);
-      expect(articles).toBeSortedBy('created_at', {
-        descending: true,
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("responds with an array of articles in descending date order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log(body)
+        let articles = body;
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              title: expect.any(String),
+              topic: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+        expect(articles.length).toEqual(12);
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
       });
```

Comment:
These are some really robust tests, Emma. Don't forget to remove the body when sending back the data.